kinto/tests/test_default_bucket.py

```diff
@@ -13,11 +11,7 @@ class DefaultBucketViewTest(BaseWebTest, unittest.TestCase):
     def test_default_bucket_exists_and_has_user_id(self):
         bucket = self.app.get(self.bucket_url, headers=self.headers)
         result = bucket.json
-        settings = self.app.app.registry.settings
-        hmac_secret = settings['cliquet.userid_hmac_secret']
-        bucket_id = hmac_digest(hmac_secret, self.principal)[:32]
```

Comment:
What I liked with this was the fact to check that a bucket_id is related to the connected user.